app/components/DR/CasesStatistics.js

```diff
@@ -16,11 +19,17 @@ class CasesStatistics extends React.Component {
       deaths: 0,
       recovered: 0,
       current: 0,
+      date: undefined,
```

Comment:
Not a good idea to use undefined as default value, always use the same type of value will should get the state